database/migrations/2024_04_23_092705_create_users_table.php

```diff
@@ -17,8 +17,8 @@ public function up(): void
             $table->string('password',255);
             $table->enum('type',['Admin','Student']);
             $table->string('avatar',255);
-            $table->string('google_token',50);
-            $table->unsignedBigInteger('student_id');
+            $table->string('google_token',50)->nullable();
```

Comment:
Không được phép sửa migration. Nếu làm trong 1 team sẽ bị ảnh hưởng toàn bộ hệ thống.
Nếu muốn thay đổi cần tạo 1 migration mới để sửa

Reply:
https://www.youtube.com/watch?v=u86VdOc3asA)](https://www.youtube.com/watch?v=u86VdOc3asA